extensions.json

```diff
@@ -180,5 +180,19 @@
     "sha256sum": "90af3124c1a68ab5b9b5d3faa480d1c77c0dd902ac1c7a71dea3f445c2df7939",
     "foxe": "https://github.com/NVIDIA-ISAAC-ROS/.github/raw/main/resources/foxglove_plugins/nvidia.nvblox_foxglove-2.1.0.foxe",
     "keywords": ["ros", "nvidia", "nvblox", "isaac"]
+  },
+  {
+    "id": "rogyken.joint-state-publisher",
+    "name": "Joint State Publisher",
+    "description": "Publishing joint state messages that can be interacted with via the slider UI.",
+    "publisher": "rogy-ken",
+    "homepage": "https://github.com/rogy-ken/foxglove-joint-state-publisher",
+    "readme": "https://github.com/rogy-ken/foxglove-joint-state-publisher/blob/main/README.md",
+    "changelog": "https://github.com/rogy-ken/foxglove-joint-state-publisher/blob/main/CHANGELOG.md",
+    "license": "MIT",
+    "version": "0.1.0",
+    "sha256sum": "3c1ba8195e31809939c89ce73320cdc47147f012f41b050029f6aae4dabc7c93",
+    "foxe": "https://github.com/rogy-ken/foxglove-joint-state-publisher/releases/download/v0.1.0/rogyken.joint-state-publisher-0.1.0.foxe",
```

Comment:
I think this URL is wrong. It works in a browser but if you `curl` it then it does not work. See the other extension listings to a URL format example.